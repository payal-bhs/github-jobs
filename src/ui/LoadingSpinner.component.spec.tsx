import React from "react";
import { shallow } from "enzyme";
import { LoadingSpinner } from "./LoadingSpinner.component";


describe("Heading", () => {
    it("render project spinner", () => {
        const spinner = shallow(<LoadingSpinner />);
        expect(spinner).toMatchSnapshot();
    });
});