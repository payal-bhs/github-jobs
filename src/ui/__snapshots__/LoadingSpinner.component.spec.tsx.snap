// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading render project spinner 1`] = `ShallowWrapper {}`;
